src/cls/log/cls_log.cc

```diff
@@ -132,7 +132,7 @@ static int cls_log_add(cls_method_context_t hctx, bufferlist *in, bufferlist *ou
       index = entry.id;
     }
 
-    CLS_LOG(0, "storing entry at %s", index.c_str());
+    CLS_LOG(20, "storing entry at %s", index.c_str());
```

Comment:
@amitkumar50 The higher the log level, the more log messages Ceph emits. By moving the message to a higher log level, it will only get emitted if the user raises the log level to at least that number (20 in this case).

For more information, read http://docs.ceph.com/docs/master/rados/troubleshooting/log-and-debug/

Also, are you subscribed to the ceph-users mailing list? This kind of question could be answered there as well.